sys/log/full/src/log.c

```diff
@@ -83,6 +79,10 @@ log_init(void)
     rc = log_nmgr_register_group();
     SYSINIT_PANIC_ASSERT(rc == 0);
 #endif
+
+#if MYNEWT_VAL(LOG_CONSOLE)
+    log_console_init();
```

Comment:
The other types of logs needed to be defined in the app before being initialized. Only `LOG_CONSOLE` would become different now after the changes introduced by this PR. I guess if MODLOG wants to use the console to log stuff, it should be the one initializing a log and anyone using the MODLOG actually gets the console log as an effect of using the MODLOG. Just a thought though, not important enough functionally.